package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "webpack": "tsx ./development/webpack/launch.ts",
     "webpack:clearcache": "./development/clear-webpack-cache.js",
-    "postinstall": "yarn webpack:clearcache",
+    "foundryup": "tsx --tsconfig ./test/helpers/foundry/tsconfig.json ./test/helpers/foundry/foundryup.mts",
+    "postinstall": "yarn webpack:clearcache && yarn foundryup --binaries anvil --version \"$(node -e \"process.stdout.write(require('./package.json').foundry)\")\"",
```

Comment:
This is another place I use the `package.json`'s `foundry` property (the other was in the circleci yaml). Have you thought of a better way to do this yet?

Reply:
Should we hardcode checksums of the binaries we expect somewhere in our repo, and then verify that the binaries are correct. `foundryup` (both this script or the original bash script), doesn't have the version immutability guarantee that `npm`/`yarn` do 🤔, so it makes our developers (and thus users, by a stretch) susceptible to supply chain attacks. 


cc @kumavis @naugtur: a summary: this PR adds foundry's `anvil` to the extension; it is a binary downloaded from Foundry's Github Releases page, so it can be swapped out at any time.

Reply:
How important is keping it up to date? Could we install it once and keep the binary in the repo? It's easier and less likely to be ignored. If we do checksums, people will just update checksums and be annoyed ;)

Reply:
> How important is keping it up to date?

This is to replace Ganache, a package that hasn't been updated in over a year now. So apparently we don't need it to be updated all too often. :-)